etp-db/src/main/sql/migration/repeatable/r-energiatodistus-tila.sql

```diff
@@ -0,0 +1,24 @@
+insert into energiatodistustila (id, name)
+values
+(0, 'luonnos'),
```

Comment:
Vanhentunut on myös yksi tila, tosin sen varmaan näkee aikaleimasta? Eli kaikki muut tilat on eksplisiittisesti tässä, paitsi vanhentunut joka on implisiittinen?

Reply:
juuri näin, käyttöliittymän näkökulmasta vanhentunut voidaan esittää tilana, mutta tietomallissa se päätellään vanhenemisajasta